WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -891,6 +883,15 @@ - (BlogDetailsSection *)publishTypeSectionViewModel
                                                  callback:^{
         [weakSelf showMediaLibraryFromSource:BlogDetailsNavigationSourceRow];
                                                  }]];
+    
+    BlogDetailsRow *pagesRow = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Site Pages", @"Noun. Title. Links to the blog's Pages screen.")
+                                             accessibilityIdentifier:@"Site Pages Row"
+                                                    image:[UIImage gridiconOfType:GridiconTypePages]
+                                                 callback:^{
+        [weakSelf showPageListFromSource:BlogDetailsNavigationSourceRow];
+                                                 }];
+    pagesRow.quickStartIdentifier = QuickStartTourElementPages;
+    [rows addObject:pagesRow];
```

Comment:
I like how the `pagesRow` is defined, then added to `rows` 👍   
I think defining `postsRow`, `mediaRow`, and `commentsRow` would make this method more readable ✨

Reply:
Great suggestion! Done in 1c7524965b367628e300b2503a8f76e4136f2e81